src/lib/modules/executor/executor.go

```diff
@@ -0,0 +1,105 @@
+package executor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/bench-routes/bench-routes/tsdb/file"
+	"github.com/go-ping/ping"
+)
+
+type Ping struct {
+	Min time.Duration `json:"minVal"`
+	Max time.Duration `json:"maxVal"`
+	Avg time.Duration `json:"avgVal"`
+}
+
+type Jitter struct {
+	Min time.Duration `json:"minVal"`
+	Max time.Duration `json:"maxVal"`
+	Avg time.Duration `json:"avgVal"`
+}
+
+type Monitor struct {
+	ResDelay  time.Duration `json:"resDelay"`
+	ResLength int64         `json:"resLength"`
+}
+
+// ExecuteMachine executes ping and jitter operations
+func ExecuteMachine(app file.Appendable, host string) error {
+	pinger, err := ping.NewPinger(host)
+	if err != nil {
+		return fmt.Errorf("error creating ping : %w", err)
+	}
+	pinger.Count = 5
+	var lastTime time.Duration
+	var sum time.Duration
+	var minJitter time.Duration = time.Hour * 24
+	var maxJitter time.Duration = 0
```

Comment:
Please group them into single `var` and remove the min, max thing from jitter as discussed.